packages/zoe/test/unitTests/test-zoe.js

```diff
@@ -134,7 +134,10 @@ test(`E(zoe).getPublicFacet - no instance`, async t => {
   // @ts-expect-error deliberate invalid arguments for testing
   await t.throwsAsync(() => E(zoe).getPublicFacet(), {
     message:
-      /In "getPublicFacet" method of \(ZoeService\): arg 0: .*"\[undefined\]" - Must be a remotable/,
+      // Golden test uses RegExp "Or" pattern to tolerate earlier versions of
+      // Endo.
+      // TODO: Remove first disjunct once no longer needed
```

Comment:
how will we know when this is no longer needed? is there a ticket? it should point here or this comment should point to it.

Reply:
@erights Does the need for the disjunct vanish due to the sync? I suspect we can get rid of it in this PR since syncing Endo eliminates the prior behavior, at least for the purposes of tests.

Reply:
Yes.

Reply:
Done.